TESTS/integration/COMMON/mbed_app.json

```diff
@@ -0,0 +1,279 @@
+            "wizfi310.event-queue-size"                 : 1024,
+            "wizfi310.provide-default"                  : true
+        },
+        "NUCLEO_L496ZG": {
+            "target.macros_remove"                      : ["MBED_TICKLESS"],
+            "target.components_add"                     : ["SD"],
+            "sd.SPI_MOSI"                               : "D11",
+            "sd.SPI_MISO"                               : "D12",
+            "sd.SPI_CLK"                                : "D13",
+            "sd.SPI_CS"                                 : "D10",
+            "target.components_add"                     : ["WIFI_WIZFI310"],
+            "target.network-default-interface-type"     : "WIFI",
+            "wizfi310.tx"                               : "D1",
+            "wizfi310.rx"                               : "D0",
+            "wizfi310.rts"                              : "PB_1",
+            "wizfi310.cts"                              : "PB_13",
+            "wizfi310.rx-buffer-size"                   : 512,
+            "wizfi310.stacksize"                        : 1536,
+            "wizfi310.event-queue-size"                 : 1024,
+            "wizfi310.provide-default"                  : true
+        },
+        "DISCO_L475VG_IOT01A": {
+            "target.macros_remove"                      : ["MBED_TICKLESS"],
+            "target.components_add"                     : ["QSPIF"],
+            "target.components_add"                     : ["WIFI_ISM43362"],
+            "target.network-default-interface-type"     : "WIFI"
+        },
+        "DISCO_L496AG": {
+            "target.macros_remove"                      : ["MBED_TICKLESS"],
+            "target.components_add"                     : ["QSPIF"],
+            "target.components_add"                     : ["WIFI_WIZFI310"],
+            "target.network-default-interface-type"     : "WIFI",
+            "wizfi310.tx"                               : "PB_6",
+            "wizfi310.rx"                               : "PG_10",
+            "wizfi310.rts"                              : "PG_12",
+            "wizfi310.cts"                              : "PG_11",
+            "wizfi310.rx-buffer-size"                   : 512,
+            "wizfi310.stacksize"                        : 1536,
+            "wizfi310.event-queue-size"                 : 1024,
+            "wizfi310.provide-default"                  : true
+        },
+        "DISCO_F413ZH": {
+            "target.components_add"                     : ["QSPIF"],
+            "target.components_add"                     : ["WIFI_ISM43362"],
+            "target.network-default-interface-type"     : "WIFI"
+        },
+        "DISCO_F469NI": {
+            "target.components_add"                     : ["QSPIF"],
+            "target.components_add"                     : ["WIFI_WIZFI310"],
+            "target.network-default-interface-type"     : "WIFI",
```

Comment:
Let's remove configurations that require a shield for now ;-)

Reply:
why remove configurations that require a shield?  That eliminates many choices.

Reply:
Shield support packages that bring tests or enable tests that exist within the OS might be nice 😉

Reply:
I believe the default configuration inside Mbed OS should contain what the platform provides, but this shouldn't stop anyone from having a custom/extended configuration to test any shield/connectivity.
Maybe we can move these custom configurations to different json file?

Reply:
Yes. For those who use a well specified connector it might be convenient to have the concept of that connector within the os.

Customization beyond a standard connector could be kept in a "target_extension.json" file.

@mark-edgeworth

Reply:
fixed